src/cabinUt/cabinUt.utils.ts

```diff
@@ -28,3 +32,11 @@ export function getVisbookId(url: string): number {
 
   return 0;
 }
+
+export function isReservationsUrl(url: string): boolean {
+  if (url.includes('reservations')) {
```

Comment:
You could parse the URL and check if the hostname contains reservations using `new URL(url)`

Reply:
Great idea, I improved the function and included more relevant tests as well. These should cover the cases that I've seen so far for different cabins.